data/seeds/014-email_addresses.js

```diff
@@ -0,0 +1,26 @@
+const email_addresses = [
+  {
+    recipient_id: 'ec6f7c16-d064-4ae3-a759-4fea047f2c76',
+    email_address: 'aoeu@gmail.com',
+    email_address_description: 'primary',
+  },
+  {
+    recipient_id: 'ec6f7c16-d064-4ae3-a759-4fea047f2c76',
+    email_address: 'aoeu@fastmail.com',
+    email_address_description: 'secondary',
```

Comment:
This works! The idea for having is as "description" was if they needed to add something like "Bob's case worker".